packages/tocco-ui/src/Icon/example.js

```diff
@@ -9,8 +9,8 @@ export default () => {
   return (
     <div>
       {/* start example */}
-      <Icon icon="address-card" /><br />
-      <Icon icon={['far', 'address-card']} /><br />
+      <Icon icon="address-card" /><Icon icon="fab, microsoft"/><br />
```

Comment:
Define as array and use curly braces like in the following line.

Reply:
As i said above this is something i explicitly want to support so i want to show it in the example